service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterMapper.java

```diff
@@ -8,14 +8,22 @@
 import java.util.Optional;
```

Comment:
!!! This file does not compile: there are duplicated imports that cause compilation error

Reply:
This error was caused by merge conflict.